include/net/nrf_cloud_location.h

```diff
@@ -41,6 +45,12 @@ enum nrf_cloud_location_type {
 	LOCATION_TYPE__INVALID
 };
 
+/** @brief Anchor name list node, to be used with @ref single-linked-list_apis */
```

Comment:
This @ref makes the doc build fail.

Reply:
removed the ref.
is there a proper way to do such a reference?

Reply:
Where exactly is this link supposed to link to? Seems that links within the API doc to structures and members work ok.